spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /show' do
+    let(:user) { User.create!(name: 'Test User', bio: 'Test bio.') }
+    it 'displays the profile page of another user' do
+      other_user = User.create!(name: 'Other User', bio: 'Bio for Other User')
+      get user_path(other_user)
+      expect(response).to have_http_status(200)
+      expect(response.body).to include(other_user.name)
+      expect(response.body).to include(other_user.bio)
+    end
+  end
+
+  describe 'Users GET /users#index' do
+    it 'checks if response status was correct' do
+      get users_path, headers: { 'HTTP_ACCEPT' => 'application/json' }
+
+      expect(response).to have_http_status(:success)
+    end
```

Comment:
* Same thing here also 
```
Make sure that for each action you check:
If the response status was correct.
If a correct template was rendered.
If the response body includes the correct placeholder text.
```
* Kindly ensure you write at least 3 tests that cover all scenarios stated above for each action